doc/COMPILING/COMPILING-MSYS.md

```diff
@@ -80,7 +80,7 @@ git clone https://github.com/CleverRaven/Cataclysm-DDA.git ./Cataclysm-DDA
 
 ```bash
 cd Cataclysm-DDA
-make -j$((`nproc`+0)) CCACHE=1 RELEASE=1 MSYS2=1 DYNAMIC_LINKING=1 SDL=1 TILES=1 SOUND=1 LOCALIZE=1 LANGUAGES=all LINTJSON=0 ASTYLE=0 TESTS=0
+PATH="/ucrt64/bin:$PATH" make -j$((`nproc`+0)) CCACHE=1 RELEASE=1 MSYS2=1 DYNAMIC_LINKING=1 SDL=1 TILES=1 SOUND=1 LOCALIZE=1 LANGUAGES=all LINTJSON=0 ASTYLE=0 TESTS=0
```

Comment:
There are multiple environments in MSYS2 (see https://www.msys2.org/docs/environments/) and `ucrt64` could be a wrong choice.

Anyway, something is not right in your MSYS2 installation if your environment does not contain proper paths and a simple call to `make` or `g++` does not work.

Reply:
@ZhilkinSerg , indeed you are right, is specificaly choose `ucrt64` here. 

Others may not hit the same problem as me, and I was able to workaround easily anyways.

I guess we can close this PR.